cluster/charts/rook-ceph-cluster/values.yaml

```
@@ -88,7 +88,7 @@ cephClusterSpec:
     # port: 8443
     # serve the dashboard using SSL
     ssl: true
-  network:
+  network: {}
```

How about we comment it out instead? Then it's consistent to uncomment lines if you want to apply the settings in the section, instead of needing to delete the `{}`
```suggestion
  #network: 
```

---

done.